tests/kafkatest/services/kafka/kafka.py

```diff
@@ -160,10 +210,10 @@ def verify_reassign_partitions(self, reassignment):
 
         # create command
         cmd = "echo %s > %s && " % (json_str, json_file)
-        cmd += "/opt/kafka/bin/kafka-reassign-partitions.sh "\
-                "--zookeeper %(zk_connect)s "\
-                "--reassignment-json-file %(reassignment_file)s "\
-                "--verify" % {'zk_connect': self.zk.connect_setting(),
+        cmd += "/opt/%s/bin/kafka-reassign-partitions.sh " % kafka_dir(node)
+        cmd += "--zookeeper %(zk_connect)s "
+        cmd += "--reassignment-json-file %(reassignment_file)s "
+        cmd += "--verify" % {'zk_connect': self.zk.connect_setting(),
```

Comment:
This looks wrong -- these are all `+=` operators and then the substitution only applies to the `"--verify"` string and doesn't do anything. I think we need to stick to either constructing the the whole template string then doing keyword-based string substitution or to substituting everything as we go.